README.md

```diff
@@ -144,13 +143,13 @@ automation:
   action:
   - service: persistent_notification.create
     data_template:
-      title: Birthday!
-      message: '{{ state_attr(''sensor.next_birthday'', ''birthday_message'') }}'
```

Comment:
This should be left as is in master-branch (so that it supports new template style in HA)